datumaro/components/progress_reporting.py

```diff
@@ -12,9 +12,9 @@
 class ProgressReporter:
     """
     Only one set of methods must be called:
-    - start - report_status - finish
-    - iter
-    - split
+        - start - report_status - finish
+        - iter
+        - split
```

Comment:
I can see the same situation in `ExactMerge`, but we don't have tabs there. What is the difference?

Reply:
I added an indentation to `ExactMerge`